lntest/itest/lnd_channel-funding_fund_max_test.go

```diff
@@ -110,7 +116,7 @@ func testChannelFundMax(networkHarness *lntest.NetworkHarness,
 		{
 			name: "wallet amount > max chan size, " +
 				"push amount == max-chan-size",
-			amount:             20000000,
+			amount:             20_000_000,
```

Comment:
Same here, please squash these changes with the commit that introduced the code.

Reply:
Done